src/dispatch/__init__.py

```diff
@@ -380,29 +296,29 @@ def get_arguments():
         "--latest",
         action="store_true",
         help=f"""{FormatText.bold}Mutually exclusive with respect to --file and --cmd.{FormatText.end}
-        Report latest jobs from {LATEST_TASKS_FILE}.""",
+        Report latest jobs from {dispatch_globals.LATEST_TASKS_FILE}.""",
     )
     tasks_source.add_argument(
         "-f",
         "--file",
         help=f"""{FormatText.bold}Mutually exclusive with respect to --latest and --cmd.{FormatText.end}
-        Specify a different location than the default {DEFAULT_TASKS_FILE} of the file containing request_id's, artifact URLs or request URLs.""",
+        Specify a different location than the default {dispatch_globals.DEFAULT_TASKS_FILE} of the file containing request_id's, artifact URLs or request URLs.""",
     )
     tasks_source.add_argument(
         "-c",
         "--cmd",
-        nargs="+",
+        action="append",
```

Comment:
Could you please reference the change in the [README.md](https://github.com/danmyway/tesar/blob/devel/README.md) and [CHANGELOG.md](https://github.com/danmyway/tesar/blob/devel/CHANGELOG.md) files?

Reply:
```
You can specify a different path to the file with -f/--file or pass the jobs to get report for straight to the commandline with -c/--cmd.
The tool is able to parse and report for multiple variants of values as long as they are separated by a new-line (in the files) or a whitespace (on the commandline).
```
(from the [README](https://github.com/danmyway/tesar/blob/devel/README.md))
Should I do the same change for `-f` option?

Reply:
Done